templates/actors/tabs/character-bastion.hbs

```diff
@@ -99,7 +102,7 @@
     </div>
     {{/if}}
     <div class="facility-header">
-        <div class="name-stacked">
+        <div class="name-stacked" data-action="useFacility">
```

Comment:
The entire `facility-header` section highlights on hover, but only the name portion actually does something when clicked. Should probably adjust the hover CSS to only apply to the name.

The highlight is also applied to a non-editable actor where clicking does nothing.